doc/background.rst

```diff
@@ -57,25 +61,100 @@ full-featured dashboards. The libraries in this ecosystem include:
    user-relevant parameters, making it simple to work with widgets
    inside and outside of a notebook context
 
-These projects can be used separately or together in a wide variety of
-different configurations to address different needs. For instance, if we
-focus on the needs of a data scientist/analyst who wants to understand
-the properties of their data, we can compare that to the approach
-suggested for a software developer wanting to build a highly custom
-software application for data of different sizes:
+
+Solving problems using PyViz tools
+----------------------------------
+
+Why do we say that PyViz is an ecosystem, as opposed to an application
+or a tool? The answer is that PyViz consists of a large set of loosely
+related, separately maintained components, not some single monolithic
+application. In practice, different people will use different PyViz
+tools in different ways to solve different problems.
+
+For instance, if we focus on the needs of a data scientist/analyst who
+wants to understand the properties of their data, we can compare that
+to the approach suggested for a software developer wanting to build a
+highly custom software application:
 
 .. image:: assets/ds_hv_bokeh.png
     :height: 220px
 
-Here Datashader is used to make large datasets practical by rendering
-images outside the browser, either directly for a programmer or via a
-convenient high-level interface using HoloViews (or hvPlot), and the
-results can be embedded in interactive Bokeh plots if desired, either
-as a static HTML plot, in a Jupyter notebook, or as a standaline
-application.
+Here both users are using Datashader, but in very different ways.  The
```

Comment:
I think if you weren't already familiar with these tools this section would be hard to follow. Maybe try to emphasize more the differences between the approaches. Also the image above doesn't show hvplot.